src/modules/auth/AuthServices/Auth.js

```diff
@@ -3,36 +3,27 @@ import PropTypes from 'prop-types'
 
 import Data from '../Data'
 import EmailPw from './EmailPw'
-import { CardForm, Row, Link, Mini, Redirect, OauthButton } from 'instruments'
+import { CardForm, Row, Link, Mini, OauthButton } from 'instruments'
 import s from './styles.scss'
 
 class Auth extends React.Component {
   render() {
-    const { authConfig = {}, login: login2, cli, pathname, token } = this.props
+    const { authConfig = {}, login: login2, cli, token } = this.props
```

Comment:
```suggestion
    const { authConfig = {}, login: showFields, cli, token } = this.props
```

I think. `login` property is true when this component is showing the login tab (vs the signup tab)

Reply:
This might not be the right loging, but that is the meaning of `this.props.login` so we don't need to do the url test.

Reply:
Yeah I thought that might be the case, but that login prop returned true for me regardless of the page 🤔. Let me play around with it a bit.

Reply:
I have a suspicion it had to do with the forced `auth` redirect that I've removed.

Reply:
Oh just chatting to greg about the `/auth` situation.